nexus/src/external_api/http_entrypoints.rs

```diff
@@ -3445,7 +3449,7 @@ async fn sagas_get_saga(
     path = "/users",
     tags = ["silos"],
 }]
-async fn silo_users_get(
+async fn silo_list_users(
```

Comment:
If we decide to go for `/users` I think this would make more sense as `users_list`. Given SDKs and CLIs will be shaped by what's generated here that would make more sense.

Reply:
Yeah, this is probably better than my suggestion of changing the path. The real issue was just the inconsistency.

Reply:
I don't feel so firmly that I believe this PR should be held up for this. The topic of API organization may warrant its own RFD.